### PR TITLE
Rename getDeviceSize to getViewportSize; add the docs

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @AlexanderMoskovkin @helen-dikareva @VasilyStrelyaev

### CODE REVIEW

File: src/api/is-valid-device-name.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
-import getDeviceSize from './get-device-size';
+import getViewportSize from './get-viewport-size';
 
 
 /**
  * Checks if the provided string is a valid device name contained in the screen size database.
  * @function
  * @name isValidDeviceName
  * @param {string} inputString - The string to be validated.
+ * @returns {boolean} value indicating that the specified string is a valid device name
```
Comment (VasilyStrelyaev) [inline src/api/is-valid-device-name.js]: ` * @returns {boolean} 'true' if the specified string is a valid device name.`

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by helen-dikareva** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**